<launch>
  <arg name="ns" default="myrobot"/>
  <arg name="id_subsystem" default="133"/>
  <arg name="id_node" default="64"/>
  <group ns="$(arg ns)">
      <node name="iop_primitive_driver" pkg="fkie_iop_component" type="iop_component">
        <param name="capability_group" value="iop_bridge"/>
        <rosparam param="services">
          [
            fkie_iop_primitive_driver: "PrimitiveDriver",
            fkie_iop_discovery: "DiscoveryClient",
            fkie_iop_accesscontrol: "AccessControl",
            fkie_iop_events: "Events",
            fkie_iop_events: "EventsClient",
            fkie_iop_transport: "Transport",
          ]
        </rosparam>
        <param name="iop_address" value="$(arg id_subsystem).$(arg id_node).33"/>
        <param name="enable_ros_interface" value="false"/>
        <param name="max_linear_x" value="3.0"/>
        <param name="max_angular_z" value="1.5"/>
        <remap from="odom" to="/odom"/>
        <remap from="cmd_vel" to="cmd_vel"/>
    </node>
  </group>
</launch>
